### DISCUSSION TITLE
Possibility to add a folder to IMPORTABLE EXTERNPROTO

### DISCUSSION THREAD

**Discussion by BenjaminDeleze** [cyberbotics/webots#5190] [Ideas]
When testing the OSM_importer, I had to add every single car PROTO one by one in the IMPORTABLE list.
This is not very convenient. 
I would have prefer to add the full vehicle folder to the list.

**Comment by ad-daniel** [comment]
This should no longer be a problem when we add a supervisor function that allows to do the IMPORTABLE declarations programmatically. When we have this function, we only need to change the controller of SUMO so that it declares what it needs to
